ui/lib/replication/addon/routes/application.js

```diff
@@ -15,9 +15,8 @@ export default Route.extend(ClusterRoute, {
         return this._super(...arguments);
       });
   },
-
   model() {
-    return this.store.findRecord('cluster', 'vault');
+    return this.store.peekRecord('cluster', 1);
```

Comment:
this one change is what makes it so the replication engine receives updated information off the cluster